WebNetSample/appsettings.json

```diff
@@ -5,6 +5,9 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "ConnectionStrings": {
+    //"WebNetSampleConnectionString": "Server=localhost;Database=WebNetSampleDB;User Id=sa;Password=qweASDzxc123+;"
```

Comment:
Actually, I have 2 computers, one is mac and the other one is windows so that I have 2 configurations because I couldn't be sure about if I'll use it again on my other computer or not. Therefore, I decided not to delete this. Should I still delete?

Reply:
Oh, I see your point now. Well, in this way, it's better to not comment it in app settings, but declare them both and add one more param specifying connection type - `docker` or `local`. On app start you can easily determine what appropriate string should be used then.